### PR TITLE
Improve usage of event names to avoid typos

### PR DESCRIPTION
**Pull request by ThimDeveloper**
### Description:
**Housekeeping and refactoring**
Common strings as identifiers to emit and subscribe to events can be error prone due to typos. This PR aims to unify all web sockets events and channels so that the risk for typos is mitigated. By also moving the event names into their own respective class we can utilize IDE suggestions for both event names and channel names. Some extra boy-scouting and housekeeping was also done.

https://user-images.githubusercontent.com/55190001/153220249-07a7bda9-f904-4184-b3f8-8884f38c4864.mov

#### Summary:
- Moved data parsers from websocket files to separate `data_parsers.py` file to declutter implementation
- Moved all constants into own `constants.py` file
- Updated all usages and imports throughout the project, including tests and examples 

### Breaking changes:
- [ ]

### New features:
- [ ]

### Fixes:
- [Unified all websocket events and subscribable channels as constants ]

### PR status:
- [ ] Version bumped
- [ ] Change-log updated

### CODE REVIEW

File: bfxapi/examples/ws/cancel_order.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import os
 import sys
+from tkinter import E
```
Comment (itsdeka) [inline bfxapi/examples/ws/cancel_order.py:3]: Hello, for what is this used?

**Comment by itsdeka** [comment]
Hello @ThimDeveloper, I like this idea and appreciate your effort.
Thank you very much.
Could you also please update the version according to semver rules?

File: bfxapi/examples/ws/cancel_order.py (hunk 1 above)
Comment (ThimDeveloper) [reply bfxapi/examples/ws/cancel_order.py:3]: Oh no! That looks like a runaway import! I will clean up both my PRs ASAP :)

**Comment by ThimDeveloper** [comment]
Hey @itsdeka !
I will update the version and change log when my other PR has been merged as it will bump the package to 2.0.2. 
This will effectively impact how I will version this PR and the order of the change log. 
:D

**Pull request closed without merging**